kasa/iot/iotdevice.py

```diff
@@ -300,7 +300,7 @@ async def update(self, update_children: bool = True) -> None:
         # If this is the initial update, check only for the sysinfo
         # This is necessary as some devices crash on unexpected modules
         # See #105, #120, #161
-        if self._last_update is None:
+        if not self._last_update:
```

Comment:
I don't recall the specific, but do you recall if there was some reason to have this as `None` when unitialized?

Reply:
I can’t recall and it’s inconsistent with `smart`. I checked everywhere and can’t see why it would ever be {} except at initialisation.